src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -26,9 +26,9 @@ public static List<String> printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         return words.stream()
-                .sorted()
-                .distinct()
                 .filter(word -> word.length() > 12)
+                .distinct()
+                .sorted((s1, s2) -> Integer.compare(s2.length(), s1.length()))
```

Comment:
sorted 함수에 Comparator를 전달해주면 되기 때문에 아래와 같은 코드도 사용이 가능할 것 같아요!! 한 번 참고해보시면 좋을 것 같습니다 :)

```suggestion
                .sorted(Comparator.comparing(String::length))
```